weave/trace_server/calls_query_builder.py

```diff
@@ -522,6 +538,11 @@ def _as_sql_base_format(
     "wb_run_id": CallsMergedAggField(field="wb_run_id", agg_fn="any"),
     "deleted_at": CallsMergedAggField(field="deleted_at", agg_fn="any"),
     "display_name": CallsMergedAggField(field="display_name", agg_fn="argMaxMerge"),
+    # constructed fields
+    "latency": ConstructedSelectField(
+        field="latency",
+        expression="dateDiff('millisecond', any(calls_merged.started_at), any(calls_merged.ended_at)) AS latency",
+    ),
```

Comment:
how do I actually do this? This is just a gross hack obviously... Sort of a new paradigm of having fields generated in select statements? unless I am misunderstanding this query construction flow.